src/test/kotlin/com/github/julianhille/sqldelight/jscipherdriver/JsTransacterTest.kt

```diff
@@ -0,0 +1,212 @@
+package com.github.julianhille.sqldelight.drivers.jscipher
+
+import com.squareup.sqldelight.Transacter
+import com.squareup.sqldelight.TransacterImpl
+import com.squareup.sqldelight.db.SqlDriver
+import kotlin.test.AfterTest
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFailsWith
+import kotlin.test.assertTrue
+import kotlin.test.fail
+
+class JsTransacterTest {
```

Comment:
Just have a cohesive naming pattern as in`JsDriverTest.kt`, please and rename all functions.